web/src/message_edit.ts

```diff
@@ -477,6 +477,14 @@ function handle_inline_topic_edit_change(this: HTMLInputElement): void {
         // the error. So, we re-enable the save button and remove the tooltip.
         ui_util.enable_element_and_remove_tooltip($topic_edit_save_button);
     }
+
+    if (!realm.realm_mandatory_topics) {
+        const $topic_not_mandatory_placeholder = $("#inline-topic-edit-placeholder");
+        $topic_not_mandatory_placeholder.css(
```

Comment:
Can this work by toggling classes? Style-wise, we try to avoid doing manual `.css()` things.

Reply:
Added 
```css
.inline-topic-edit-placeholder-visible {
    visibility: visible;
}
```
and updated the logic to `addClass` or `removeClass` as required.

Reply:
I don't see that in what is pushed?